src/unleash-proxy.ts

```diff
@@ -44,11 +57,83 @@ export default class UnleashProxy {
         this.middleware = router;
 
         // Routes
-        router.get('/health', this.health.bind(this));
-        router.get('/', this.getEnabledToggles.bind(this));
-        router.post('/', this.lookupToggles.bind(this));
-        router.post('/client/metrics', this.registerMetrics.bind(this));
-        router.get('/client/features', this.unleashApi.bind(this));
+        router.get(
+            '/health',
+            openApiService.validPath({
+                responses: withStandardResponses(200, 503)(),
+            }),
+            this.health.bind(this),
+        );
+
+        router.get(
+            '/',
+            openApiService.validPath({
+                parameters: [
+                    ['appName', "Your application's name"],
+                    ['userId', "The current user's ID"],
+                    ['sessionId', "The current session's ID"],
+                    ['remoteAddress', "Your application's IP address"],
+                    ['properties', 'Additional properties'],
+                ].map(([k, v]) => ({
+                    name: k,
+                    description: v,
+                    in: 'query',
+                })),
```

Comment:
Could perhaps be a helper, like withStandardResponses.

```ts
openApiService.validPath({
    parameters: createRequestParameters({
        appName: "Your application's name",
        userId: "The current user's ID",
        sessionId: "The current session's ID",
        remoteAddress: "Your application's IP address",
        properties: 'Additional properties',
    })
});

Reply:
Aye! I agree.

Reply:
Interesting! Looks like there's a way to spec arbitrary params, though I haven't tried it:

https://stackoverflow.com/questions/49582559/how-to-document-dynamic-query-parameter-names-in-openapi-swagger

If that doesn't work, maybe we could use the description field here as well.

Reply:
Works!